app/views/comments/_edit.html.erb

```diff
@@ -22,20 +22,20 @@
 
         <p>
           <span id="c<%= comment.id%>uploading" class="uploading" style="display:none;">
-            <%= t('comments._edit.uploading') %>
+            <%= translation('comments._edit.uploading') %>
```

Comment:
Nice! The function name is `translations` - plural - so if you could get that changed then we'll be all set!